### PR TITLE
Add technical charter. Closes #81.

### LINKED ISSUE #81: Add a Governance section to the README and upload the Project Charter
Per additional Linux Foundation guidance.

### PR DESCRIPTION
**Pull request by krook**
Signed-off-by: Daniel Krook <krook@us.ibm.com>

**Describe at a high level the solution you're providing**
Uploaded the Technical Charter document to supplement the contribution guidelines.

**Is this a patch, a minor version change, or a major version change**
Minor addition, no code.

**Is this related to an open issue?**
#81

### CODE REVIEW

**Comment by krook** [comment]
I also ran a "format Markdown" command in VSCode, which is why you see some spacing changes.

**Review by Gaurav-Ramakrishna — APPROVED** [review]
Thank you

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,27 +2,30 @@
...
+- Localization and improvisation of the application
+- Add/Update crop data
+- Enhance watering advise agorithm
+- Pilot testing the solution
+- Improve the documentation
+- Report bug and/or work on existing [issues](https://github.com/Call-for-Code/Liquid-Prep/issues)
+- Request/recommend new features in our [issues](https://github.com/Call-for-Code/Liquid-Prep/issues) or [discussions board](https://github.com/Call-for-Code/Liquid-Prep/discussions)
 
-Please read our [contributing guidelines](#contributing-guidelines).
+Please read our [contributing guidelines](#contributing-guidelines) and founding [Technical Charter](Liquid-Prep-Technical-Charter.pdf).
```
Comment (Gaurav-Ramakrishna) [inline CONTRIBUTING.md:14]: @krook is the Technical Charter PDF uploaded in the project? Gives me a 404 error.

Comment (krook) [reply CONTRIBUTING.md:14]: Ah. Forgot to include the PDF itself. The review process proves its value!

Comment (krook) [reply CONTRIBUTING.md:14]: Uploaded in the latest commit. Should be good now.

Comment (Gaurav-Ramakrishna) [reply CONTRIBUTING.md:14]: Awesome thank you

**Review by Gaurav-Ramakrishna — APPROVED** [review]

**Pull request merged**